### PR TITLE
[data] skip running against nightly arrow

### PR DESCRIPTION
**Pull request by aslonnie**
it is currently consistently failing, and I do not think it can go back without any action.

### CODE REVIEW

**Comment by aslonnie** [comment]
you can flip it back when it is fixed; based on my shallow understanding, the arrow interface has changed, and it won't be fixed unless we change the code on our part.

**Comment by aslonnie** [comment]
I do not mind running it again and and again on postmerge (and hoping it can show some differnet reult?) but it is currently polluting the banner with all the errors like this:

image

**Comment by can-anyscale** [comment]
I'll leave this up for data folks ;)

File: .buildkite/data.rayci.yml [hunk 1]
```diff
@@ -54,10 +54,9 @@ steps:
     tags: 
       - python
       - data
+      - disabled # TODO(bveeramani): remove skipping when nightly tests are stable
     # run only on postmerge
     if: build.env("BUILDKITE_PIPELINE_ID") == "0189e759-8c96-4302-b6b5-b4274406bf89"
```
[the remark this answers is not in the document]
Comment (aslonnie) [reply .buildkite/data.rayci.yml:59]: it is already skipping on premerge (based on the `if` condition).

I am just a bit annoyed about the error printing.

I can also just add an `exit 1` in the commands.. if people want to see a fainted red box there.

[the remark this answers is not in the document]
Comment (can-anyscale) [reply .buildkite/data.rayci.yml:59]: right, I just mean to replace the if condition with `skip-on-premerge`.

the `disabled` tag I'll leave it to data team ;)

[the remark this answers is not in the document]
Comment (aslonnie) [reply .buildkite/data.rayci.yml]: done. replaced with `skip-on-premerge`.

rayci magic~

**Comment by pcmoritz** [comment]
Feel free to merge this until https://github.com/ray-project/ray/issues/41139 is fixed but I do think we should make sure to fix that in a timely manner -- it will only get worse as time goes on (and more regressions might pile up).

**Comment by aslonnie** [comment]
> Feel free to merge this until #41139 is fixed but I do think we should make sure to fix that in a timely manner -- it will only get worse as time goes on (and more regressions might pile up).

totally. I am trusting that part is on @bveeramani and @anyscalesam :)

**Comment by bveeramani** [comment]
The error should be fixed by https://github.com/ray-project/ray/pull/41036. Let's try to get the PR merged tomorrow, and then we don't need to skip the tests?

**Pull request closed without merging**